compiler/src/dmd/cli.d

```diff
@@ -910,45 +911,64 @@ dmd -cov -unittest myprog.d
     ];
 
     /// Returns all available previews
     static immutable previews = [
+        // Note: generally changelog entries should be added to the spec
         Feature("dip25", "useDIP25",
-            "implement https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md (Sealed references)", true, true),
+            "implement Sealed References DIP",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md", true, true),
         Feature("dip1000", "useDIP1000",
-            "implement https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1000.md (Scoped Pointers)"),
+            "implement Scoped Pointers DIP",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1000.md"),
         Feature("dip1008", "ehnogc",
-            "implement https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1008.md (@nogc Throwable)"),
+            "implement @nogc Throwable DIP",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1008.md"),
         Feature("dip1021", "useDIP1021",
-            "implement https://github.com/dlang/DIPs/blob/master/DIPs/accepted/DIP1021.md (Mutable function arguments)"),
-        Feature("bitfields", "bitfields", "add bitfields https://github.com/dlang/dlang.org/pull/3190"),
-        Feature("fieldwise", "fieldwise", "use fieldwise comparisons for struct equality"),
+            "implement Mutable Function Arguments DIP",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/accepted/DIP1021.md"),
+        Feature("bitfields", "bitfields", "add C-like bitfields",
+            "https://github.com/dlang/dlang.org/pull/3190"),
+        Feature("fieldwise", "fieldwise", "use fieldwise comparisons for struct equality",
+            "https://dlang.org/changelog/2.085.0.html#no-cmpsb"),
         Feature("fixAliasThis", "fixAliasThis",
-            "when a symbol is resolved, check alias this scope before going to upper scopes"),
+            "when a symbol is resolved, check alias this scope before going to upper scopes",
+            "https://github.com/dlang/dmd/pull/8885"),
         Feature("intpromote", "fix16997",
-            "fix integral promotions for unary + - ~ operators", false, true),
+            "fix integral promotions for unary + - ~ operators",
+            "https://dlang.org/changelog/2.078.0.html#fix16997", false, true),
         Feature("dtorfields", "dtorFields",
-            "destruct fields of partially constructed objects", false, false),
+            "destruct fields of partially constructed objects",
+            "https://dlang.org/changelog/2.098.0.html#dtorfileds", false, false),
         Feature("rvaluerefparam", "rvalueRefParam",
-            "enable rvalue arguments to ref parameters"),
+            "enable rvalue arguments to ref parameters",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/rejected/DIP1016.md"),
```

Comment:
I think that is the DIP that Walter is referring to here: https://github.com/dlang/dmd/pull/9817

Given that is marked rejected, perhaps we should add `deprecated: true` for this option.

Reply:
It actually implements Andrei's reboot presented at DConf 2019: 
https://gist.github.com/andralex/e5405a5d773f07f73196c05f8339435a

Reply:
```suggestion
            "https://gist.github.com/andralex/e5405a5d773f07f73196c05f8339435a"),
```